ruby-challenges.rb

```diff
@@ -32,3 +51,15 @@
 # Expected output: 'LEARN is not a palindrome'
 palindrome_tester3 = 'Rotator'
 # Expected output: 'Rotator is a palindrome'
+
+ def palindrome(string)
+    if string.downcase == string.downcase.reverse
+        'Palindrome'
+    else 
+        'Not a palindrome'
+    end
+end
+
+puts palindrome(palindrome_tester1.to_s)
+puts palindrome(palindrome_tester2)
+puts palindrome(palindrome_tester3)
```

Comment:
Nicely done, but don't forget to compare your output to the expected output, which is asking for the string repeated in the output!